80s Game/Assets/Scripts/Multiplayer/PlayerInputWrapper.cs

```diff
@@ -16,10 +16,13 @@ public class PlayerInputWrapper : MonoBehaviour
     private PlayerInput playerInput;
     private List<Joycon> joycons;
     private float joyconSensitivyAdjust = 5.0f;
+    private bool fireHeld = false;
+    private float fireDelay = 0.2f;
```

Comment:
We may wanna make this value configurable down the line for easy balancing purposes